libagent/device/trezor.py

```diff
@@ -32,6 +37,61 @@ def _is_open_tty(stream):
     return not stream.closed and os.isatty(stream.fileno())
 
 
+def _pin_communicate(self, program, message, error=None, options={}):
+    args = [program]
+    if 'DISPLAY' in os.environ:
+        args.extend(['--display', os.environ['DISPLAY']])
+    try:
+        entry = subprocess.Popen(
+            args,
+            encoding='utf-8',
+            stdin=subprocess.PIPE, stdout=subprocess.PIPE,
+        )
+    except OSError as e:
```

Comment:
This is Py2 specific.  How would you handle compatibility in this case?  Catch both?  Wrap subprocess?